src/hooks/use-import-export.tsx

```diff
@@ -84,15 +84,24 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 				},
 			} ) );
 
-			const handleImportError = async ( error?: unknown ) => {
-				await getIpcApi().showErrorMessageBox( {
-					title: __( 'Failed importing site' ),
-					message: __(
-						'An error occurred while importing the site. Verify the file is a valid Jetpack backup, Local, Playground, .wpress or .sql database file and try again. If this problem persists, please contact support.'
-					),
-					error,
-					showOpenLogs: true,
-				} );
+			const handleImportError = async ( error: unknown ) => {
+				if ( error instanceof Error && error.message.includes( 'absolute path' ) ) {
+					await getIpcApi().showErrorMessageBox( {
+						title: __( 'Failed importing site' ),
+						message: __(
+							'The ZIP archive is incorrect. Try to unpack and pack it again. If this problem persists, please contact support.'
```

Comment:
```suggestion
							'The ZIP archive is invalid. Try to unpack and pack it again. If this problem persists, please contact support.'
```
I think "invalid" makes more sense than "incorrect" in this context.